src/Compilers/Core/Portable/Diagnostic/DiagnosticInfo.cs

```diff
@@ -138,6 +138,8 @@ internal DiagnosticInfo GetInstanceWithSeverity(DiagnosticSeverity severity)
 
         #region Serialization
 
+        bool IObjectWritable.ShouldReuseInSerialization => true;
```

Comment:
>true [](start = 59, length = 4)

Why do we have to reuse instances of this type?

Reply:
It was effectively reusable before. 
I do not see how it would ever be reused though. Will change to `false`

---
In reply to: [181826677](https://github.com/dotnet/roslyn/pull/26156#discussion_r181826677) [](ancestors = 181826677)